### PR TITLE
chore: update to 0.9.3

### PR DESCRIPTION
**Pull request by weichweich**
## fixes KILTprotocol/ticket/issues/1358

- fixes KILTprotocol/ticket/issues/1357

## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: nodes/standalone/src/service.rs [hunk 1]
```diff
@@ -43,27 +43,22 @@ native_executor_instance!(
...
 	sp_consensus::DefaultImportQueue<Block, FullClient>,
 	sc_transaction_pool::FullPool<Block, FullClient>,
 	(
-		sc_consensus_aura::AuraBlockImport<
-			Block,
-			FullClient,
-			sc_finality_grandpa::GrandpaBlockImport<FullBackend, Block, FullClient, FullSelectChain>,
-			AuraPair,
-		>,
+		sc_finality_grandpa::GrandpaBlockImport<FullBackend, Block, FullClient, FullSelectChain>,
```
Comment (wischli) [inline nodes/standalone/src/service.rs:53]: Any reason why we switch to GrandPa from AuRa in the standalone runtime? Shouldn't we stick with the same consensus?

**Review by wischli — COMMENTED** [review]
I have one question regarding switching the consensus for standalone.

File: nodes/standalone/src/service.rs (hunk 1 above)
Comment (weichweich) [reply nodes/standalone/src/service.rs:53]: We don't switch the consensus. We still use Aura+Grandpa. [Aura is just not needed for the block import anymore](https://github.com/paritytech/substrate/pull/8881).

**Review by wischli — APPROVED** [review]
LGTM! Thanks for clarifying my question.

**Pull request merged**